README.md

```diff
@@ -1,2 +1,62 @@
-# PIVX Shielding
-WASM library for PIVX shielding
+# PIVX Shield
+WASM library for interoperation with the PIVX Shield protocol.
+
+## Getting started
+### Compile
+To compile the non-multicore version,
+```bash
+wasm-pack build --target web
+```
+To compile with multicore support enabled, make sure the nightly compiler and `rust-src` are installed. Version 2023-03-28 is recommended.
+```bash
+./build_multicore.sh
+```
+This will generate the two versions in `pkg/` and `pkg_multicore/`. Then, run `npm i /path/to/this/project/js/`, to install the javascript wrapper in your project.
+
+### Examples
+
+```js
+import { PIVXShield as Shield } from "pivx-shield-js";
+const shield = await Shield.create({
+	seed: yourSeed,
```

Comment:
what is yourSeed?

Reply:
We don't provide a function to create a seed, so it's a variable that the user must provide

Reply:
I meant explain better what is represents